src/main/java/org/jenkinsci/test/acceptance/po/UpdateCenter.java

```diff
@@ -71,7 +70,8 @@ public boolean isInstalled(String pluginShortName) throws InstallationFailedExce
      */
     public boolean waitForInstallationToComplete(final PluginSpec... specs) throws InstallationFailedException {
         open();
-        waitFor(driver, not(Matchers.hasContent("Pending")), 60); // Wait for all plugins to get installed
+        // Wait for all plugins to get installed
+        waitFor(driver, not(Matchers.hasElement(by.xpath("//*[@id='log']//*[contains(.,'Pending') or contains(.,'Installing')]"))), 60);
```

Comment:
Why was the matcher for hasContent("Pending") causing a race condition? How is that different from asserting the xpath element that contains Pending?